### PR TITLE
feat: add network parameters and description of max bytes

### LINKED ISSUE #79: Document that PFD endpoint accepts up to 2 MB under default settings
The Node `submit_pfd` API endpoint must indicate it can accept up-to 1 MB size for a data submission

### PR DESCRIPTION
**Pull request by jcstein**
### [PREVIEW](https://celestiaorg.github.io/docs/pr-preview/pr-1350/nodes/mainnet#network-parameters)

Resolves #79 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: nodes/mainnet.md [hunk 1]
```diff
@@ -33,6 +33,49 @@ functional, users may encounter occasional instability or reduced performance.
 
 <MainnetVersionTags/>
 
+## Network parameters
+
+Full network parameters, such as [max bytes](https://github.com/celestiaorg/celestia-app/blob/23d13d4de41631dc3c52f7d94fc214e44d03962d/specs/src/specs/params.md?plain=1#L30),
+can be found in the
+[celestia-app specifications](https://celestiaorg.github.io/celestia-app/specs/params.html)
+or from `max_bytes` in
+[Mainnet Beta validator's consensus parameters](https://rpc.lunaroasis.net/consensus_params).
```
Comment (rootulp) [inline nodes/mainnet.md:42]: [clarification] there are more network parameters than consensus parameters so the second part of this sentence seems a bit misleading. Proposal to either remove "Full" from the beginning or the last phrase which links to consensus params.

```suggestion
Full network parameters, such as [max bytes](https://github.com/celestiaorg/celestia-app/blob/23d13d4de41631dc3c52f7d94fc214e44d03962d/specs/src/specs/params.md?plain=1#L30),
can be found in the
[celestia-app specifications](https://celestiaorg.github.io/celestia-app/specs/params.html).
```

File: nodes/mainnet.md [hunk 2]
```diff
@@ -33,6 +33,49 @@ functional, users may encounter occasional instability or reduced performance.
...
+$\text{Total Bytes} = (1 \times 478 \, \text{bytes}) + (4094 \times 482 \, \text{bytes}) = 1,973,786 \, \text{bytes}$
+<!-- markdownlint-enable MD013 -->
+
+Please note that there isn't a precise upper bound on the maximum total
+blob size. It depends on several factors:
+
+- The maximum square size, which is determined by a governance parameter
+and a versioned constant.
+- The maximum bytes in a block, which is influenced by a governance parameter
+and a hard-coded constant in CometBFT.
```
Comment (rootulp) [inline nodes/mainnet.md:74]: [nit]

```suggestion
- The maximum bytes in a block, which is determined by a governance parameter
and a hard-coded constant in CometBFT.
```

File: nodes/mainnet.md [hunk 3]
```diff
@@ -33,6 +33,49 @@ functional, users may encounter occasional instability or reduced performance.
...
+- The amount of shares occupied by the PFB transaction share.
+
+These factors can cause the maximum total blob size to vary.
```
Comment (rootulp) [inline nodes/mainnet.md:77]: [optional]

```suggestion
These factors can cause the maximum total blob size that can be included in one block to vary.
```

**Review by rootulp — APPROVED** [review]
Thanks for this!

File: nodes/mainnet.md (hunk 1 above)
Comment (jcstein) [reply nodes/mainnet.md:42]: thanks for the review! good call. I am going to stick with this suggestion, as it simplifies things for the reader.

**Pull request merged**